scripts/jdg-debc

```diff
@@ -13,12 +13,17 @@ fi
 
 CHANGES=$(find . -maxdepth 1 -name \*.changes ! -name \*_source.changes)
 
+HOST_ARCH="$(dpkg-architecture -qDEB_HOST_ARCH)"
```

Comment:
Do you have any objections against using:

  HOST_ARCH="$(dpkg --print-architecture)"

instead here too? (See commit 59e1a99c797c1cb7bf6ae44d8504bdf7ab0d9a11 for details).